tailwind-plugins/button.js

```diff
@@ -104,7 +104,7 @@ module.exports = [
           "&:focus, &.focus": {
             boxShadow: "0 0 0 0.2rem rgb(130 138 145 / 50%)",
           },
-          '&[href*="//"]:not([href*="foundation.mozilla.org"]):not([href*="donate.mozilla.org"])':
+          '&[href*="//"]:not([href*="www.mozillafoundation.org"]):not([href*="donate.mozilla.org"])':
```

Comment:
```suggestion
          '&[href*="//"]:not([href*="mozillafoundation.org"]):not([href*="donate.mozilla.org"])':
```

Hi @robdivincenzo! I noticed that this js file is applying CSS styles to buttons that have links which include `foundation.mozilla.org` (going to be `mozillafoundation.org` soon) somewhere in the HREF. 

Since this is a sort of "wildcard" type of rule that is meant to capture any button linking to `mozillafoundation.org`, would it be best to remove the `www` from here? 

My thinking is that CMS users might not always add the `www` to the HREF, which would cause those links to be ignored.